source/agora/test/NetworkDiscovery.d

```diff
@@ -20,7 +20,10 @@ import agora.test.Base;
 ///
 unittest
 {
-    TestConf conf = TestConf.init;
+    TestConf conf =
+    {
+        retry_delay : 100.msecs,
+    };
```

Comment:
Why ?

Reply:
The first network test in this module will fail after the default `retry_delay` is set to `500.msecs` because the test checks the `NodeInfo` of all the nodes immediately.